up42/http/session.py

```diff
@@ -16,14 +19,19 @@ def request(self, *args, **kwargs) -> requests.Response:
         return response
 
 
-@functools.lru_cache
 def create(
-    create_adapter=http_adapter.create,
-    create_auth=oauth.Up42Auth,
+    auth: requests.auth.AuthBase,
+    create_adapter: HttpAdapterFactory = http_adapter.create,
+    version: str = utils.get_up42_py_version(),
 ) -> requests.Session:
     session = StatusValidatingSession()
     adapter = create_adapter()
     for schema in SCHEMAS:
         session.mount(schema + "://", adapter)
-    session.auth = create_auth(create_adapter=create_adapter)
+    session.auth = auth
+    session.headers = {
+        "Content-Type": "application/json",
+        "cache-control": "no-cache",
+        "User-Agent": f"up42-py/{version} ({REPOSITORY_URL})",
```

Comment:
do we need repo URL in the headers? I can't think of a use-case where we might be needing this.
If not, let's use this as an opportunity to drop it.

Reply:
@seedlit I'm not sure whether this will affect our BI or not. Let's add this as a ticket for @PelJon to confirm first